React/Views/RCTMapManager.m

```diff
@@ -41,6 +41,7 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(maxDelta, CGFloat)
 RCT_EXPORT_VIEW_PROPERTY(minDelta, CGFloat)
 RCT_EXPORT_VIEW_PROPERTY(legalLabelInsets, UIEdgeInsets)
+RCT_REMAP_VIEW_PROPERTY(mapType, mapType, MKMapType)
```

Comment:
@paramaggarwal I didn't realize that. Updated to use ` RCT_EXPORT_VIEW_PROPERTY`.